drag_and_drop_v2/translations/en/LC_MESSAGES/text.po

```diff
@@ -0,0 +1,168 @@
+msgid "The Top Zone"
+msgstr ""
+
+#: default_data.py
+msgid "The Middle Zone"
+msgstr ""
+
+#: default_data.py
+msgid "The Bottom Zone"
+msgstr ""
+
+#: default_data.py
+msgid "Use this zone to associate an item with the top layer of the triangle."
+msgstr ""
+
+#: default_data.py
+msgid ""
+"Use this zone to associate an item with the middle layer of the triangle."
+msgstr ""
+
+#: default_data.py
+msgid ""
+"Use this zone to associate an item with the bottom layer of the triangle."
+msgstr ""
+
+#: default_data.py
+#, python-brace-format
+msgid "Correct! This one belongs to {zone}."
+msgstr ""
+
+#: default_data.py
+msgid "No, this item does not belong here. Try again."
+msgstr ""
+
+#: default_data.py
+msgid "You silly, there are no zones for this one."
+msgstr ""
+
+#: default_data.py
+msgid "Drag the items onto the image above."
+msgstr ""
+
+#: default_data.py
+msgid "Good work! You have completed this drag and drop problem."
+msgstr ""
+
+#: default_data.py
+msgid "Goes to the top"
+msgstr ""
+
+#: default_data.py
+msgid "Goes to the middle"
+msgstr ""
+
+#: default_data.py
+msgid "Goes to the bottom"
+msgstr ""
+
+#: default_data.py
+msgid "I don't belong anywhere"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Title"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid ""
+"The title of the drag and drop problem. The title is displayed to learners."
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Drag and Drop"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Show title"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Display the title to the learner?"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Problem text"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "The description of the problem or instructions shown to the learner"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Show \"Problem\" heading"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Display the heading \"Problem\" above the problem text?"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Weight"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "The maximum score the learner can receive for the problem"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Item background color"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "The background color of draggable items in the problem."
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Item text color"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Text color to use for draggable items."
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Problem data"
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid ""
+"Information about zones, items, feedback, and background image for this "
+"problem. This information is derived from the input that a course author "
+"provides via the interactive editor when configuring the problem."
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid ""
+"Information about current positions of items that a learner has dropped on "
+"the target image."
+msgstr ""
+
+#: drag_and_drop_v2.py
+msgid "Indicates whether a learner has completed the problem at least once"
+msgstr ""
```

Comment:
@asadiqbal08 I noticed that this file does not contain entries for strings from `js_templates.html`. The templates in this file are Handlebars templates, and they use non-standard markup for strings that should be translated (see [here](https://github.com/edx-solutions/xblock-drag-and-drop-v2/blob/master/drag_and_drop_v2/templates/html/js_templates.html#L15) for an example). That's why the strings they contain are not being added to the translation file.

You didn't specify how you are generating translation files, but maybe you could try and see if you can modify the `makemessages` command to pick up strings from `js_templates.html`? See [here](https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#customizing-makemessages) for more info. Having translations available for these strings would be helpful for course authors.